float64 x_acc
float64 y_acc
float64 z_acc
float64 x_rot
float64 y_rot
float64 z_rot
